### PR TITLE
Removed tests.py and added individual tests

### PR DESCRIPTION
**Pull request by DanWaldman**
*this PR is dependent on PR #72 Because it needs to have changes in user, so merge #72 first*
in this PR we:
- resolved issues: #74 #75 #76 
- removed tests.py.
- made new file - test_user.py with all user tests (containing fixtures and in requested format).
- made new file - test_professor.py with all professor tests (containing fixtures and in requested format).
- made new file - test_university.py with all university tests (containing fixtures and in requested format).
- all tests passed.

### CODE REVIEW

File: AcadeMeData/test_professor.py
Comment (Omeramsc) [inline AcadeMeData/test_professor.py:18]: Since `generate_university` is used in both the professor and university tests, I'd add it to a [conftest](https://docs.pytest.org/en/6.2.x/fixture.html#scope-sharing-fixtures-across-classes-modules-packages-or-session) file

File: AcadeMeData/models.py
Comment (Omeramsc) [inline AcadeMeData/models.py:33]: Isn't that the same change made in a former PR? 
if this PR depends on it, mention it in the description. we wouldn't want to accidentally merge this before the former.

**Review by Omeramsc — CHANGES_REQUESTED** [review]
Left some comments
also, link an issue to the PR.

File: AcadeMeData/models.py
Comment (DanWaldman) [reply AcadeMeData/models.py:33]: Mentioned it, thanks

File: AcadeMeData/test_professor.py
Comment (DanWaldman) [reply AcadeMeData/test_professor.py:18]: will do

**Comment by Yarboa** [comment]
> * resolved issues: [Remove tests.py and add test_user for user tests #74](https://github.com/redhat-beyond/AcadeMe/issues/74) [add test_university - for university class tests #75](https://github.com/redhat-beyond/AcadeMe/issues/75) [add test_professor - for professor class tests #76](https://github.com/redhat-beyond/AcadeMe/issues/76)

I do not think it is related, 
Who ever merged first, will be merged, in case tests will fail after merge rebase it will be fixed


> * removed tests.py.

File: AcadeMeData/test_professor.py
Comment (DanWaldman) [reply AcadeMeData/test_professor.py]: > Since `generate_university` is used in both the professor and university tests, I'd add it to a [conftest](https://docs.pytest.org/en/6.2.x/fixture.html#scope-sharing-fixtures-across-classes-modules-packages-or-session) file

If you could take a look - I tried adding content.py and it didn't work, looked online but couldn't fix the error.

File: AcadeMeData/conftest.py
Comment (Yarboa) [reply AcadeMeData/conftest.py:5]: If you are using fixture, 
and you restrict it to session, you have to call other two db fixtures, which are django fixtures
you can look here 
pipenv run pytest --fixtures /vagrant/AcadeMeData/

please refer this:

 @pytest.fixture(scope="session")
-def generate_university(university_id=5, name='The Technion', location="Haifa",
+def generate_university(django_db_setup, django_db_blocker, university_id=5, name='The Technion', location="Haifa",
                         description="Best University in Israel"):
-    university = University(university_id=university_id, name=name, location=location,
-                            description=description)
-    university.save()
+    with django_db_blocker.unblock():
+        university = University(university_id=university_id, name=name, location=location,
+                                description=description)
+        university.save()
     return university

Comment (DanWaldman) [reply AcadeMeData/conftest.py]: Just removed the scope, thanks

**Review by Yarboa — APPROVED** [review]
LGTM

**Review by DanielItkes — APPROVED** [review]

**Review by bregman-arie — APPROVED** [review]

**Review by shakedhagag — APPROVED** [review]
Everything looks good.
just a question what does the change in pytest.ini does?

**Pull request merged**
